### PR TITLE
Add Kudobuzz detection

### PR DESCRIPTION
**Pull request by khempenius**
website:
https://kudobuzz.com/
https://github.com/kudobuzz

examples:
https://www.theringerhoops.com/
https://www.ericraviliousprints.com/
https://www.sayitwithflowers.uk.com/
https://www.thepureindianstore.com/
https://www.snogear.com/

### CODE REVIEW

File: src/technologies/k.json [hunk 1]
```diff
@@ -1056,6 +1056,26 @@
...
+    "cats": [
+      5
+    ],
+    "description": "Kudobuzz is a tool for collecting and managing customer reviews.",
+    "dom": {
+      "div[id*='kudobuzz_']": {
+        "exists": ""
+      }
+    },
+    "icon": "Kudobuzz.png",
```
Comment (nurbek91) [inline src/technologies/k.json:1069]: Kudobuzz
32x32 svg

File: src/technologies/k.json [hunk 2]
```diff
@@ -1056,6 +1056,26 @@
...
+  "Kudobuzz": {
+    "cats": [
+      5
+    ],
+    "description": "Kudobuzz is a tool for collecting and managing customer reviews.",
+    "dom": {
+      "div[id*='kudobuzz_']": {
+        "exists": ""
+      }
+    },
```
Comment (nurbek91) [inline src/technologies/k.json:1068]: we can remove this trace due redundancy

File: src/technologies/k.json [hunk 3]
```diff
@@ -1056,6 +1056,26 @@
...
+    "icon": "Kudobuzz.png",
+    "implies": [
+      "Svelte"
+    ],
+    "js": {
+      "Kudos": ""
+    },
```
Comment (nurbek91) [inline src/technologies/k.json:1075]: for complete certainty
```
"js": {
      "Kudos.apiServer": "api\\.kudobuzz\\.com"
    },
```

File: src/technologies/k.json [hunk 4]
```diff
@@ -1056,6 +1056,26 @@
     "icon": "Kubernetes.svg",
     "website": "https://kubernetes.io/"
   },
+  "Kudobuzz": {
+    "cats": [
+      5
```
Comment (nurbek91) [inline src/technologies/k.json:1061]: 90 reviews category

**Review by nurbek91 — CHANGES_REQUESTED** [review]

**Review by nurbek91 — APPROVED** [review]

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**